Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN apt-get update && \
     pip3 install --no-cache-dir -r requirements.txt
 
 # Copy python scripts and entrypoint
-COPY *.py .
+COPY *.py ./
```

Comment:
Must explicitly indicate the destination is a directory by having the suffix `/`